esp/bindings/SOAP/Platform/soapbind.cpp

```diff
@@ -164,7 +153,10 @@ int CHttpSoapBinding::onSoapRequest(CHttpRequest* request, CHttpResponse* respon
         catch (IMultiException* mex)
         {
             StringBuffer ns;
-            soapFault.setown(makeSoapFault(request,mex, generateNamespace(*request->queryContext(), request, request->queryServiceName(), request->queryServiceMethod(), ns).str()));
+            StringBuffer schemaLocation;
+            generateNamespace(*request->queryContext(), request, request->queryServiceName(), request->queryServiceMethod(), ns);
+            getSchemaLocation(*ctx, request, ns, schemaLocation);
+            soapFault.setown(makeSoapFault(request,mex, ns.str(), schemaLocation.str()));
```

Comment:
@asselitx does scapps always return the schemaLocation in a SoapFault?

Reply:
Yes